src/cmd-fetch

```diff
@@ -8,7 +8,7 @@ dn=$(dirname "$0")
 FILE=cache/pkgcache-repo
 if [ -d "${FILE}" ]
 then
```

Comment:
optional: some context here in a comment on why we run it with `sudo` might be nice

Reply:
It's hidden out of the default context lines view in GitHub, but this is just above the `sudo cosa prune --pkgcache` call. Was hoping the consistent use of `sudo` there was enough. I can add a comment in a follow-up if you'd like.

Reply:
whatever you think is appropriate. feel free to merge here.